app/upgrades/types.go

```diff
@@ -26,12 +26,15 @@ type Upgrade struct {
 	UpgradeName string
 
 	// CreateUpgradeHandler defines the function that creates an upgrade handler
-	CreateUpgradeHandler func(*module.Manager, module.Configurator, *keepers.AppKeepers) upgradetypes.UpgradeHandler
+	CreateUpgradeHandler CreateUpgradeHandler
```

Comment:
No strong opinion but I'm a bit hesitant on using the same name for variable name and type

Reply:
update type to UpgradeHandlerCreator in https://github.com/babylonlabs-io/babylon/pull/113/commits/42b62c2c0dc8be6637c90164955f0b4031e4baf9